benchmarking/commons/benchmark_definitions/fcnet.py

```diff
@@ -17,7 +17,7 @@
 
 def fcnet_benchmark(dataset_name):
     return SurrogateBenchmarkDefinition(
-        max_wallclock_time=1200,
+        max_wallclock_time=3600,
```

Comment:
Also part of earlier PR, hope to get merged before. Increases time for fcnet, which is a bit short.